### PR TITLE
feat(manager-components): build and publish manager components

### PR DESCRIPTION
**Pull request by aboungnaseng-ovhcloud**
ref: MANAGER-14822

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | #14822
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Version the entire library

### CODE REVIEW

**Review by antleblanc — DISMISSED** [review]
> Some checks were not successful
Could you please have a look?

There's also an empty commit `3f016f39e85b212bc1f8ae3ec957db6908e1c75a`.

Looks like there's some duplicates with #12865.

**Comment by aboungnaseng-ovhcloud** [comment]
> > Some checks were not successful
> > Could you please have a look?
> 
> There's also an empty commit `3f016f39e85b212bc1f8ae3ec957db6908e1c75a`.
> 
> Looks like there's some duplicates with #12865.

 #12865 This PR is for bumping the version of ODS in 18

**Review by anooparveti — APPROVED** [review]

**Review by chipp972 — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by chipp972 — APPROVED** [review]

File: packages/manager-react-components/vite.config.mts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    rollupOptions: {
+      external: [
+        'react',
+        'react-dom',
+        '@tanstack/react-query',
+        'react-router-dom',
+        'react-i18next',
+        'i18next',
+        'i18next-http-backend',
+        '@ovh-ux/manager-react-shell-client',
```
Comment (frenautvh) [inline packages/manager-react-components/vite.config.mts:44]: i'm not sure that you want to bundle @ovh-ux/manager-core-api or zustand in the dist

you could mark those dependencies as external

Comment (frenautvh) [reply packages/manager-react-components/vite.config.mts:44]: same for tanstack *

Comment (aboungnaseng-ovhcloud) [reply packages/manager-react-components/vite.config.mts:44]: We have a ticket to move it, in the next sprint

**Review by frenautvh — DISMISSED** [review]

**Review by frenautvh — APPROVED** [review]

**Review by chipp972 — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by antleblanc — CHANGES_REQUESTED** [review]
Commit `6a6de1c` (scope `ci`) is undoing changes from the previous commit.
Could you please update it and keep atomic commit changes.
Thanks

**Comment by aboungnaseng-ovhcloud** [comment]
> Commit `6a6de1c` (scope `ci`) is undoing changes from the previous commit. Could you please update it and keep atomic commit changes. Thanks

Do you mean you want to squash first and second commit ?

**Review by antleblanc — DISMISSED** [review]
Changes on `.github/workflows/run-bdd-tests.yml` looks good to me.

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**